server/food_bank_app/api/users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import UserViewSet
+
+urlpatterns = [
+  path('self-detail/', UserViewSet.as_view({'get': 'retrieve', 'post': 'create'})),
```

Comment:
we can use something like `me` or `profile`.
```suggestion
  path('profile/', UserViewSet.as_view({'get': 'retrieve', 'post': 'create'})),
```